packages/console/src/pages/Connectors/index.tsx

```diff
@@ -1,68 +1,125 @@
-import { ConnectorDTO } from '@logto/schemas';
-import React from 'react';
+import { ConnectorDTO, ConnectorType } from '@logto/schemas';
+import React, { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
-import { Link } from 'react-router-dom';
+import { useLocation } from 'react-router-dom';
 import useSWR from 'swr';
 
+import placeholder from '@/assets/images/social-connectors-placeholder.svg';
 import Button from '@/components/Button';
 import Card from '@/components/Card';
 import CardTitle from '@/components/CardTitle';
-import ImagePlaceholder from '@/components/ImagePlaceholder';
-import ItemPreview from '@/components/ItemPreview';
+import Status from '@/components/Status';
+import TabNav, { TabNavLink } from '@/components/TabNav';
 import { RequestError } from '@/swr';
 
+import ConnectorName from './components/ConnectorName';
 import * as styles from './index.module.scss';
 
 const Connectors = () => {
-  const { t, i18n } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const location = useLocation();
+  const isSocial = location.pathname === '/connectors/social';
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { data, error } = useSWR<ConnectorDTO[], RequestError>('/api/connectors');
   const isLoading = !data && !error;
 
+  const emailAndSmsConnectors: Array<[ConnectorDTO | undefined, string]> | undefined =
+    useMemo(() => {
+      if (isSocial || !data) {
+        return;
+      }
+
+      return [
+        [
+          data.find((connector) => connector.metadata.type === ConnectorType.Email),
+          t('connectors.type.email'),
+        ],
+        [
+          data.find((connector) => connector.metadata.type === ConnectorType.SMS),
+          t('connectors.type.sms'),
+        ],
```

Comment:
It seems not strait forward. Maybe this logic can be wrapped into a local component that will use this data.
please look at the comments below.